### PR TITLE
Customers can select a pickup date and time

### PR DESCRIPTION
**Pull request by ethanschreur**
**What does this PR do?**
- This pull request allows customers to select a pickup date and time in an "order detail" page

**Description of the task to be completed** 
- Create an "order detail" page
- Write helper functions to help create the calendar
- Display a calendar that covers the options for pickup dates which should be two weeks around the start of each month.
- Each day has options for AM and PM
- Customers can only select one option on the calendar

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/order_details
- View the calendar with the two weeks around the start of each month
- Make sure you can only select one option

### CODE REVIEW

File: app/helper.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    else:
+        second_month = next_month
+    return second_month
+
+def get_month_header(which, prev_month, curr_month, next_month, last_week, first_week):
+    if which == "first":
+        month_header = f"{prev_month} {last_week[0]} - {curr_month} {first_week[-1]}"
+    else:
+        month_header = f"{curr_month} {last_week[0]} - {next_month} {first_week[-1]}"
+    return month_header
```
Comment (alexkayabula) [inline app/helper.py:128]: @ethanschreur please add concise comments to all these functions describing their use in the application.

**Review by alexkayabula — CHANGES_REQUESTED** [review]

**Comment by ethanschreur** [comment]
Should be ready for review now.

**Comment by alexkayabula** [comment]
I am going to review it later on and get back to you.

**Review by alexkayabula — APPROVED** [review]
LGTM.

**Pull request merged**